src/js/routes/Welcome.jsx

```diff
@@ -60,6 +61,32 @@ export default class Intro extends Component {
     this.setState({loading: true});
   }
 
+  shareToFacebookButton () {
```

Comment:
Hi Rob,
Thank you for suggestion.
I will try to make a separate social button component so that it will become more reusable.